vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -4252,6 +4264,35 @@ unit:DeciB_M
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Decibel Referred to 1mw" ;
 .
+unit:DeciB_MilliV
+  a qudt:DimensionlessUnit ;
+  a qudt:LogarithmicUnit ;
+  a qudt:Unit ;
+  dcterms:description "<p class=\"lm-para\">It can be used to express a change in value (e.g., +1 dB or −1 dB) or an absolute value. In the latter case, it expresses the ratio of a value to a fixed reference value; when used in this way, a suffix that indicates the reference value is often appended to the decibel symbol. For example, if the reference value is 1 volt, then the suffix is \"V\" (e.g., \"20 dBV\"), and if the reference value is one milliwatt, then the suffix is \"m\" (e.g., \"20 dBm\").</p>"^^qudt:LatexString ;
```

Comment:
This description still seems illogical. 
if the reference value is 1 volt, then the suffix is \"V\" (e.g., \"20 dBV\") - OK, that sounds good.
if the reference value is one milliwatt, then the suffix is \"m\" (e.g., \"20 dBm\") - Wait! Why "m" and not "mW"?
How would we handle millivolt if "m" is used for milliwatt?